smsdk/client.py

```diff
@@ -662,13 +754,17 @@ def get_machine_names(self, source_type=None, clean_strings_out=True):
         else:
             return machines["source"].to_list()
 
-    def get_machine_types(self, source_type=None, *args, **kwargs):
+    def get_machine_types(self, *args, **kwargs):
```

Comment:
Already had this one =None

Reply:
Yes. But just for the sake of uniformity with other APIs, I kept it.